### ISSUE TITLE
Consider storing UI colors in an asset for hot reloading

### ISSUE THREAD

**Issue by MiniaczQ** [TheBevyFlock/bevy_new_2d#31]
Mentioned in #16

**Comment by benfrankel** [comment]
Waiting on BSN, I suppose. I do have an implementation of this in my own repo fwiw:

- Config: https://github.com/benfrankel/bevy_jam_template/blob/main/src/core/config.rs
- Theme config: https://github.com/benfrankel/bevy_jam_template/blob/main/src/core/theme.rs

**Comment by MiniaczQ** [comment]
Out of scope for 0.14 template like #71
